tests/backup/backup_kubevirt_test.go

```diff
@@ -141,8 +142,18 @@ var _ = Describe("{KubevirtVMBackupRestoreWithDifferentStates}", func() {
 				go func(backupName string, appCtx *scheduler.Context) {
 					defer GinkgoRecover()
 					defer wg.Done()
+					_, preRuleName, err := CreateKubevirtBackupRuleForAllVMsInNamespace(ctx, appCtx.ScheduleOptions.Namespace, "pre", "default")
+					log.FailOnError(err, "Unable to create Pre Rule")
+					log.Infof("Pre rule Name - [%s]", preRuleName)
+					preRuleUid, err := Inst().Backup.GetRuleUid(orgID, ctx, preRuleName)
+					log.FailOnError(err, "Unable fetch pre rule uid")
+					_, postRuleName, err := CreateKubevirtBackupRuleForAllVMsInNamespace(ctx, appCtx.ScheduleOptions.Namespace, "post", "default")
+					log.FailOnError(err, "Unable to create Post Rule")
+					log.Infof("Post rule Name - [%s]", postRuleName)
+					postRuleUid, err := Inst().Backup.GetRuleUid(orgID, ctx, postRuleName)
+					log.FailOnError(err, "Unable fetch post rule uid")
 					log.InfoD("creating backup [%s] in source cluster [%s] (%s), organization [%s], of namespace [%s], in backup location [%s]", backupName, SourceClusterName, sourceClusterUid, orgID, appCtx.ScheduleOptions.Namespace, backupLocationName)
-					err := CreateBackupWithValidation(ctx, backupName, SourceClusterName, backupLocationName, backupLocationUID, []*scheduler.Context{appCtx}, labelSelectors, orgID, sourceClusterUid, "", "", "", "")
+					err = CreateBackupWithValidation(ctx, backupName, SourceClusterName, backupLocationName, backupLocationUID, []*scheduler.Context{appCtx}, labelSelectors, orgID, sourceClusterUid, preRuleName, preRuleUid, postRuleName, postRuleUid)
```

Comment:
Please apply the pre and post rule to all the backups taken as part of this test case and the Upgrade test case

Reply:
Done, dashboard with run - https://aetos.pwx.purestorage.com/resultSet/testSetID/452940